### DISCUSSION TITLE
Can't upload image to prove academic status.

### DISCUSSION THREAD

**Discussion by GunwooJo** [community/community#86104] [GitHub Education] [answered]
### Hi everyone,

I tried to upload my certificate of university current enrollment image. 

Resolution is 3024 x 4032. 
Size is 2MB. 
Form is JPEG. 

When I click 'process my application' button, I only can see message 'processing information...'.
Also, I found error message in chrome devtools console like this.

Failed to load resource: the server responded with a status of 413 ()

How can I prove my academic status?
<img width="835" alt="스크린샷 2024-01-10 오후 5 11 54" src="https://github.com/community/community/assets/40854185/89a95fa3-b531-4707-b992-aeeaf293fda2">
<img width="666" alt="스크린샷 2024-01-10 오후 5 18 03" src="https://github.com/community/community/assets/40854185/e2d6876c-04d3-49c3-8dce-e3393ba35a73">

**Comment by nabilhayek** [comment, marked as answer]
You've gotten a error 413 response from the server which means that your file exceeds the server's file size limit. You can try to compress or resize the image with various online tools. Looks like your image has been taken from your phone, which usually results in a image with quite big resolution due to the megapixels in your phones camera. A 2MB image with 3024 x 4032 is probably too large for the form you're trying to submit.

**Reply by GunwooJo** [reply]
I really appreciate for your answer. The first site helps me to compress image. Thank you. Have a good day!

**Comment by naanatin** [comment]
I stumbled on this today as well. Would be more informative and user friendly if the UI would inform the user that the file size is too large instead of just leaving them waiting.

**Comment by jemshit** [comment]
My image is 870KB, but description says "between 100KB and 1MB in size"

**Comment by hammad-umt** [comment]
My Student Card Consist Valid till on the back side and i am not able to upload single picture also I don’t have access to any letter

**Comment by KarimAED** [comment]
After playing around with the image size a bunch, it seems that any image above 700KB fails without error message.
